src/arguments/song.ts

```diff
@@ -18,7 +21,14 @@ export default class extends Argument {
 		let returnAll = false;
 		let tracks: TrackData[] = [];
 		let soundcloud = true;
-		if (parsedURL) {
+
+		if (Reflect.has(message.flagArgs, 'i') || Reflect.has(message.flagArgs, 'import')) {
+			if (message.attachments.size === 0 && !arg) throw message.language.get('musicManagerImportQueueNotFound');
+			const url = message.attachments.size > 0 ? message.attachments.first()!.url : arg!;
+
+			tracks = this.filter(message, remainingUserEntries, await this.loadQueue(message, url));
```

Comment:
What happens if the URL is not a valid queue URL?

Reply:
It throwsssss... but ofc I forgot the trycatch, gimme a sec